mino/minogrpc/server.go

```diff
@@ -549,9 +547,29 @@ func (o *overlay) Join(addr, token string, certHash []byte) error {
 }
 
 func (o *overlay) makeCertificate() error {
+	hostname, err := o.myAddr.GetHostname()
+	if err != nil {
+		return xerrors.Errorf("error parsing node's hostname: %v", err)
+	}
+
+	ipAddr := net.ParseIP(hostname)
```

Comment:
we can simplify the whole by just calling once `ipAddrs, err := net.LookupIP(hostname)`